module/ptu.js

```diff
@@ -861,3 +861,33 @@ Hooks.on("preCreateItem", async function (item, data, options, sender) {
   // In preCreate[document] hook, you can update a document's data class using `document.data.update` before it is committed to the database and actually created.
   await item.data.update({ "data.origin": origin });
 });
+
+class dexButton extends Application {
```

Comment:
```suggestion
class DexButton extends Application {
```

Class names should be capitalized.